packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -67,6 +69,7 @@ typedef AutocompleteFieldViewBuilder = Widget Function(
   TextEditingController textEditingController,
   FocusNode focusNode,
   VoidCallback onFieldSubmitted,
+  GlobalKey fieldKey,
```

Comment:
Elsewhere you also used the name textFieldKey, can you make them all the same name if it makes sense?